lib/nadia/api.ex

```diff
@@ -5,7 +5,7 @@ defmodule Nadia.API do
 
   alias Nadia.Model.Error
 
-  @default_timeout 4
+  @default_timeout 8
```

Comment:
https://github.com/edgurgel/httpoison/blob/87aabe18be27bf25af2a4182b4754cd0a35aa1dd/lib/httpoison/base.ex#L127-L128

default value for recv_timeout is 5s, so it's (4 + 1) right now.